### DISCUSSION TITLE
How to use it

### DISCUSSION THREAD

**Discussion by Sherlock-feng** [emp-toolkit/emp-tool#190] [General]
I don't find any information which discribe how to use the emp-tool to generate circuits about my own c++ code. Can you tell me where is the interface or how to use the emp-tool?

**Comment by wangxiao1254** [comment]
I think you can start reading the test file https://github.com/emp-toolkit/emp-tool/blob/master/test/gen_circuit.cpp and the headers in https://github.com/emp-toolkit/emp-tool/tree/master/emp-tool/circuits
